locales/en/messages.json

```diff
@@ -5329,11 +5329,18 @@
         "message": "Overlay for the right transmitter stick position."
     },
     "osdTextElementDisplayName": {
-        "message": "Display name",
+        "message": "Display name (Pilot name)",
         "description": "One of the elements of the OSD"
     },
     "osdDescElementDisplayName": {
-        "message": "Display name as set by the \"display_name\" cli command"
+        "message": "Display name (pilot name) as set in the Configuration tab.</br>Can also be set via the \"display_name\" CLI variable."
```

Comment:
(craft name)

Reply:
I think the `osdDescElementDisplayName` message should say"
"Pilot name as set in the Configuration tab.</br>Can also be set via the \"pilot_name\" CLI variable."
We should use `Pilot name` and `pilot_name` when we mean Pilot Name.

Reply:
The `craft name` has its dedicated `craftName` / `configurationCraftNameHelp` translation keys; I left the `osdTextElementDisplayName` and `osdDescElementDisplayName` ones solely for backwards compatibility when configuring a firmware older than MSP `v1.45` (then `display name` will still be show and `pilot name` will be hidden).
The new `pilot name` also has two separate translation keys - `configurationPilotName` and `configurationPilotNameHelp`. 

I added some notes in the MR description on the current backwards compatibility behavior. Nevertheless - I'm open to any ideas how to make the whole backwards compatibility setup less confusing. :grinning: